qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test.properties

```diff
@@ -5,5 +5,6 @@ login.location=Pharmacy
 webdriver=chrome
 login.auto=true
 headless=true
+webdriver.type=remote
```

Comment:
is this change part of the ticket requirement?

Reply:
Resolved thanks.